darts/models/forecasting_model.py

```diff
@@ -604,31 +616,32 @@ def fit(self,
             ) -> None:
         """ Fits/trains the model on the provided series
 
-        Defines behavior that should happen when calling the `fit()` method of every forecasting model.
+        Defines behavior that should happen when calling the `fit()` method of every global forecasting model.
 
         Some models support training only on one time series, while others can handle a sequence.
         Similarly, some models can handle covariates.
 
-        Some covariates are know in the future, and others aren't. This is a property of the `TimeSeries`, which
+        Some covariates are known in the future, and others aren't. This is a property of the `TimeSeries`, which
```

Comment:
Actually we should remove this sentence. When writing it I had in mind to capture some information about "when a covariate is known" in the TimeSeries, but it's not done, and at the moment global models support only "historic" covariates.

Reply:
Given the upcoming PR introducing future covariates, I think we could leave this, what do you think? Unless there is an imminent release.